examples/freertos/CMakeLists.txt

```
@@ -111,6 +129,42 @@ set(INCLUDE_DIRS
     ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
+if(FREERTOS_F100RB_EXAMPLE)
+    set(TARGET_NAME ${PROJ_NAME}-f100)
```

Again, this is just to keep spirit with the other examples
```suggestion
    set(TARGET_NAME ${PROJ_NAME}-f100rb)
```